supervisor/container.go

```diff
@@ -252,3 +263,130 @@ func (c *Container) reap() {
 	os.RemoveAll(containerSharedDir)
 	os.RemoveAll(filepath.Join(c.ownerPod.sv.StateDir, c.Id))
 }
+
+func mountToRootfs(m *specs.Mount, rootfs, mountLabel string) error {
+	// TODO: we don't use mountLabel here because it looks like mountLabel is
+	// only significant when SELinux is enabled.
+	var (
+		dest = m.Destination
+	)
+	if !strings.HasPrefix(dest, rootfs) {
+		dest = filepath.Join(rootfs, dest)
+	}
+
+	switch m.Type {
+	case "proc", "sysfs", "mqueue":
+		if err := os.MkdirAll(dest, 0755); err != nil {
+			return err
+		}
+		// Selinux kernels do not support labeling of /proc or /sys
+		return mount.Mount(m.Source, dest, m.Type, strings.Join(m.Options, ","))
+	case "tmpfs":
+		stat, err := os.Stat(dest)
+		if err != nil {
+			if err := os.MkdirAll(dest, 0755); err != nil {
+				return err
+			}
+		}
+		if err := mount.Mount(m.Source, dest, m.Type, strings.Join(m.Options, ",")); err != nil {
+			return err
+		}
+		if stat != nil {
+			if err = os.Chmod(dest, stat.Mode()); err != nil {
+				return err
+			}
+		}
+		return nil
```

Comment:
proc sysfs mqueue tmpfs MUST pass into hyperstart and let hyperstart mount it.
hyperstart haven't has such abi, so they should be skipped.
Could you add a issue for such purpose?